### PR TITLE
Add plugins menu item to Tools menu.

### PR DESCRIPTION
**Pull request by srazi**
@nowrep would you please test if these commits work for you, unfortunately for me QZ crashes just like  #1719 .

### CODE REVIEW

**Comment by nowrep** [comment]
And what about fixing that crash? :P

**Comment by nowrep** [comment]
But indeed, it may crash only on Windows, because dynamic loading of libraries works slightly different than on Linux.

**Comment by srazi** [comment]
@nowrep: did you mean it works for you?

> And what about fixing that crash? :P

Well it is strange. it crashes when calling `m_submenuExtensions->clear()` I also remove parent of QActions. again deleting them leads to crash.

**Comment by nowrep** [comment]
I didn't test it yet.

File: src/lib/app/mainmenu.cpp [hunk 1]
```diff
@@ -621,6 +627,10 @@ void MainMenu::init()
     ADD_ACTION("Tools/RssReader", m_menuTools, QIcon(), tr("RSS &Reader"), SLOT(showRssManager()), "");
     ADD_ACTION("Tools/WebInspector", m_menuTools, QIcon(), tr("Web In&spector"), SLOT(toggleWebInspector()), "Ctrl+Shift+I");
     ADD_ACTION("Tools/ClearRecentHistory", m_menuTools, QIcon::fromTheme(QSL("edit-clear")), tr("Clear Recent &History"), SLOT(showClearRecentHistoryDialog()), "Ctrl+Shift+Del");
+
+    m_submenuExtensions = new QMenu(tr("&Extensions"));
```
Comment (nowrep) [inline src/lib/app/mainmenu.cpp]: Is there a leak? You always create the menu in aboutToHide, but never delete.

Comment (srazi) [inline src/lib/app/mainmenu.cpp]: No.
`m_submenuExtensions` is created in `MainMenu::init()` and its actions are created as its child therefore `m_submenuExtensions->clear()` deletes them.

Comment (nowrep) [inline src/lib/app/mainmenu.cpp]: Ah right, this is init(), i got confused with one line above (aboutToHideToolsMenu) :D

**Comment by nowrep** [comment]
Yes, works fine here.

**Comment by nowrep** [comment]
Also cherry-picked to `master`.

**Pull request merged**